### PR TITLE
[ClacoForm] Refactors ClacoForm resource with react components from core

### PR DESCRIPTION
**Pull request by kitan1982**
| Q             | A
| ------------- | ---
| Bug fix?      | no
| New feature?  | no
| BC breaks?    | no

### CODE REVIEW

**Comment by Elorfin** [comment]
You also need to make your custom CascadeSelect a `data/types` and add `meta.creatable : true`. You will be able to add it to your `fields` list and handle it with the standard `FormContainer`.

File: plugin/claco-form/Resources/modules/resources/claco-form/player/entry/components/form-field.jsx [hunk 1]
```diff
@@ -13,20 +14,33 @@ import {EmailGroup} from '#/main/core/layout/form/components/group/email-group.j
 import {DateGroup} from '#/main/core/layout/form/components/group/date-group.jsx'
 import {FileGroup} from '#/main/core/layout/form/components/group/file-group.jsx'
 
-// deprecated
-// only used by claco-form
-// todo : rewrite claco-form to use form.jsx
-export const FormField = props => {
+const FormField = props => {
```
Comment (Elorfin) [inline plugin/claco-form/Resources/modules/resources/claco-form/player/entry/components/form-field.jsx]: This component must be deleted.

Comment (kitan1982) [reply plugin/claco-form/Resources/modules/resources/claco-form/player/entry/components/form-field.jsx]: It is still in use in ClacoForm when a template is used.
However as it is only used by ClacoForm, it has been move to ClacoForm bundle.

Comment (Elorfin) [reply plugin/claco-form/Resources/modules/resources/claco-form/player/entry/components/form-field.jsx]: Nope. If you need it, you must use : https://github.com/claroline/Distribution/blob/11.x/main/core/Resources/modules/data/form/components/field.jsx

Comment (kitan1982) [reply plugin/claco-form/Resources/modules/resources/claco-form/player/entry/components/form-field.jsx]: Ok. It will be switched to that component.

**Pull request merged**